brendan-tuckerman/week7/mon/ajax-movie/css/style.css

```diff
@@ -0,0 +1,37 @@
+body{
+    
+    color: #5B58EE;
+}
+li {
+    list-style: none;
+}
+
+h1{
+    color: #8102f899;
+}
+
+
+.allMoviesList {
+    text-align: center;
+    line-height: 1.4;
+}
+
+#singleMovieDetails{
+    text-align: center;
+    line-height: 1.4;
+}
+
+.main_container{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 75vh;
```

Comment:
Not sure if you are seeing what i'm seeing, but when i search a movie, the CSS is funny, and some results are not shown visually on the page, its showing in the inspector (results are sitting outside of the page).  i had to turn off the `height: 75vh` to fix this